reflex/utils/pyi_generator.py

```diff
@@ -17,8 +17,11 @@
 from types import ModuleType, SimpleNamespace
 from typing import Any, Callable, Iterable, Type, get_args
 
-import black
-import black.mode
+try:
+    import black
+    import black.mode
+except Exception:
+    pass
```

Comment:
```suggestion
except ImportError:
    black = None
```

then down below, instead of catching blanket Exception, check for `if black is not None`